doc/sphinx-guides/source/api/native-api.rst

```diff
@@ -2992,9 +2994,60 @@ Deletes an ``AuthenticatedUser`` whose ``id``  is passed. ::
+- Remove the user's access from all datasets and files
+- Cancel any pending file access requests generated by the user
+- Remove the user from all groups
+- No longer have notifications generated or sent by the Dataverse installation
+
+Disabling a user with this endpoint will keep:
+
+- The user's contributions to datasets, including dataset creation, file uploads, publishing, submitting for review, and returning to author.
+- Any Dataverse Collections where the user is the sole administrator
+- The user's access to datafiles in the Dataverse installation, including guestbook records and acceptance of terms of use.
+- The user's account information (specifically name, email, affiliation, and position)
+
+If it is determined that disabling a user is not sufficient, it is possible anonymize a user through use of the :ref:`merge-accounts-label` feature. See :ref:`Anonymize a User <disable-a-user>` for more information about this process. 
+
+.. _anonymize-a-user:
+
+Anonymize a User
+~~~~~~~~~~~~~~~~
+
+In some cases, due to privacy laws or other organizational policies, it may be required to anonymize a user. This process should only be used in cases where a Dataverse installation has the need to completely remove the traces of a user from the Dataverse installation. This process is only provided because it is not feasible for the Dataverse Software to provide deletion and deactivation options that will satisfy all use cases and all privacy laws, so the most complete deletion option is provided here. 
+
+Note: Before going through these steps, please note that you will be permanently losing:
+
+- The user's contributions to datasets, including dataset creation, file uploads, publishing, submitting for review, and returning to author. This will all be marked as "Anonymous User" in your Dataverse installation, and it will not be possible to track who made changes as part of the archival record.
+- Records of the user's access to datafiles in the Dataverse installation, including guestbook records and acceptance of terms of use. This could potentially introduce issues where it is no longer possible for data curators and owners to follow up with those who have accessed data on the application, which may be a legal requirement or data sharing agreement. It will be up to the Dataverse installation administrators to determine which datasets the user has accessed to determine if anonymization is feasible in these situations.    
+- The user's account information (specifically name, email, affiliation, and position). This will make it impossible to follow up with the user being deleted. You may find that you need to contact a user about a security breach or something else in the future, and you will not able to do so. 
+
+To anonymize a user in your Dataverse installation:
+
+- Create a user called "Anonymous User"
+- Use the :ref:`merge-accounts-label` functionality to merge the user you wish to anonymize INTO the Anonymous User
```

Comment:
add a step to disable the user.

Reply:
done, https://github.com/IQSS/dataverse/pull/7585/commits/2af33fd9662328d86518c6f8c985c187398d1a0b